components/script/dom/websocket.rs

```diff
@@ -268,7 +268,12 @@ impl WebSocket {
             let pipeline_id = self.global().pipeline_id();
             self.global()
                 .script_chan()
-                .send(CommonScriptMsg::Task(WebSocketEvent, task, Some(pipeline_id)))
+                .send(CommonScriptMsg::Task(
+                    WebSocketEvent,
+                    task,
+                    Some(pipeline_id),
+                    TaskSourceName::Networking,
```

Comment:
Please add a TODO, and file an issue, noting that we should be using a dedicated `websocket-task-source` here, see https://html.spec.whatwg.org/multipage/web-sockets.html#websocket-task-source (I got to this by first looking up this  `BufferedAmountTask`, following the spec link, and then scrolling down until I found the words "The task source for all tasks queued in this section is the WebSocket task source.")